packages/upswyng-web/src/components/__tests__/Search.test.tsx

```diff
@@ -15,12 +15,12 @@ jest.mock("react-router-dom", () => ({
 describe("<Search/>", () => {
   const getSubject = () => shallow(<Search />);
 
-  it("renders a <SearchResults/> when provided a search query", () => {
+  xit("renders a <SearchResults/> when provided a search query", () => {
```

Comment:
is this not working?

Reply:
I don't think we should commit code with `xit`. Maybe just remove it and put a TODO? Or at least put a TODO comment and file an issue.

Reply:
I'm actually working on a separate update for the search that will repair this issue. I'll make a PR for that first, merge it in, and then catch this up so that this isn't an issue.

Reply:
But yes, I temporarily needed to comment out Search for this.

Reply:
created #278 that we can merge in first. The base problem was that `<SearchResults/>` was being shared by search and category pages. I updated the category/subcategory pages to expect `TResource` but search results don't receive the same. The linked PR will account for that.

Reply:
^^^ that PR has been merged so this is no longer an issue.